Formula/dmd.rb

```diff
@@ -105,6 +105,30 @@ def install
         inreplace "posix.mak", "install: $(TOOLS) $(CURL_TOOLS)", "install: $(TOOLS) $(ROOT)/dustmite"
         system "make", "install", *make_args
       end
+    else
+      make_args.unshift "DMD_DIR=#{buildpath}", "DRUNTIME_PATH=#{buildpath}/druntime", "PHOBOS_PATH=#{buildpath}/phobos"
+      (buildpath/"druntime").install resource("druntime")
+      (buildpath/"phobos").install resource("phobos")
+      system "make", "-C", "druntime", *make_args
+      system "make", "-C", "phobos", "VERSION=#{buildpath}/VERSION", *make_args
+
+      resource("tools").stage do
+        inreplace "posix.mak", "install: $(TOOLS) $(CURL_TOOLS)", "install: $(TOOLS) $(ROOT)/dustmite"
+        system "make", "install", *make_args
+      end
+
+      (include/"dlang/dmd").install Dir["druntime/import/*"]
+      cp_r ["phobos/std", "phobos/etc"], include/"dlang/dmd"
+      lib.install Dir["druntime/lib/*", "phobos/**/libphobos2.a"]
+
+      conf = buildpath/"dmd.conf"
+      # Can't use opt_include or opt_lib here because dmd won't have been
```

Comment:
You should be able to fix them up in `post_install`

Reply:
I'm a bit confused by this, though. Why do these paths actually need to exist already if we're just writing out a config file?

Reply:
Sorted, just using `opt_include` and `opt_lib` now (doesn't apply to stable, but soon we'll be able to delete all that)

Reply:
other upside is that it won't trigger a warning about dmd.conf being changed every time you upgrade because the paths will be the same.